raiden-ts/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - [#613] Implement waiting for confirmation blocks on on-chain transactions (configurable)
 
 ### Changed
+- [#926] Introduce loglevel logging framework (config.logger now specifies logging level)
```

Comment:
I think we need to add a legend for the issues as well, similarly to https://raw.githubusercontent.com/raiden-network/light-client/master/raiden-dapp/CHANGELOG.md